src/App.js

```diff
@@ -101,39 +92,43 @@ class App extends React.Component {
     })
   }
 
-  handleClick(e) {
+  loadHomePage() {
```

Comment:
does this need a prevent default function?

Reply:
It does not, it's called in the header along side a prevent default